plugin/unnumbered-ptp/unnumbered-ptp.go

```diff
@@ -326,19 +326,6 @@ func setupContainerVeth(netns ns.NetNS, ifName string, mtu int, hostAddrs []netl
 			return fmt.Errorf("failed to look up %q: %v", ifName, err)
 		}
 
-		if masq {
```

Comment:
I'm assuming that if there is a if , there is a place where this is setup.
A flag or something ?
We could remove that flag or use it.

Reply:
The problem is the flag is used somewhere else (where it's needed)
I should definitely remove the flag from this specific function though

Reply:
Just did that, definitely makes more sense
Merging